ios/tournenizer/tournenizer/ViewControllers/PendingTeamRequestsViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  PendingTeamRequestsViewController.swift
+//  tournenizer
+//
+//  Created by Ankush Rayabhari on 3/10/18.
+//  Copyright © 2018 Ankush Rayabhari. All rights reserved.
+//
+
+import Foundation;
+import UIKit;
+
+class PendingTeamRequestsViewController : UIViewController {
+    let teamRequestView = TeamRequestListViewController();
+    var team: Team!;
+
+    var contentView: UIView!;
+    var backView: UIButton!;
+    var selectLabel: UILabel!;
+    var topBackBackground: UIView!;
+
+    let actionsBarHeight: CGFloat = 35;
+    let topTitlePadding: CGFloat = 20;
+    let sideTitlePadding: CGFloat = 15;
+    let buttonPadding: CGFloat = 10;
+    let iconSize: CGFloat = 25;
+    let selectText = "User Team Requests";
```

Comment:
Why are you calling it User Team Requests? Call it something understandable like just Team Requests